rpcserver.go

```diff
@@ -258,6 +258,11 @@ func calculateFeeRate(satPerByte, satPerVByte uint64, targetConf uint32,
 
 }
 
+// GetAllPermissions returns all the permissions required to interact with lnd.
+func GetAllPermissions() ([]bakery.Op, []bakery.Op) {
```

Comment:
I know we've been adding code in Pool and other repos for exporting this as well. But instead of exporting the individual permission components that exist, couldn't we just iterate over the map returned by `MainRPCServerPermissions()` and see what permissions are _actually used_ by any of the RPC methods?

Reply:
Sure that sounds good to me. Loop over them in the Terminal code rather than here in lnd?

Reply:
Sounds good, took a stab at that